src/pages/_app.tsx

```diff
@@ -1,23 +1,46 @@
 import "@/styles/globals.css";
 import type { AppProps } from "next/app";
-import { Inter, Staatliches } from "@next/font/google";
-import { QueryClient, useQuery } from "@tanstack/react-query";
+import { QueryClient } from "@tanstack/react-query";
 import { useState } from "react";
-import { ReactQueryDevtools } from "@tanstack/react-query-devtools";
 import { SessionProvider } from "next-auth/react";
-import { QueryClientProvider } from "@tanstack/react-query";
-const queryClient = new QueryClient();
+import { useStyleStore, useSzieStore } from "./stores/styleStore";
+import clsx from "clsx";
+import { useEffect } from "react";
 
 export default function App({ Component, pageProps }: AppProps) {
-  const [queryClient] = useState(() => new QueryClient());
+  const { darkMode } = useStyleStore();
+  const { size } = useSzieStore();
+
+  useEffect(() => {
+    const htmlElement = document.getElementsByTagName("html")[0];
```

Comment:
lets only grab the htmlElement from the DOM once, when the component has been initialised, i.e. bring this out one layer